### ISSUE TITLE
Does WinAppDriver support Appium PushFile and PullFile ?

### ISSUE THREAD

**Issue by mike140588** [microsoft/WinAppDriver#898]
Using the Appium dot net driver plug in to Visual Studio 2019,  the following c# results in a WinAppDriver unknown command error on the machine under test:

Session.PushFile("C:\\temp\\hello.txt", "hello");

![screenshot](https://user-images.githubusercontent.com/56118736/66127293-37527b00-e5e3-11e9-92b2-58b8f1877c7e.png)

**Comment by licanhua** [comment]
I don't think so. [SupportedAPIs.md](https://github.com/microsoft/WinAppDriver/blob/master/Docs/SupportedAPIs.md) only supports Appium/app/launch and app/close

**Comment by hassanuz** [comment]
WinAppDriver does not support this at this time.
